Gruntfile.js

```diff
@@ -139,8 +139,7 @@ module.exports = grunt => {
       options: {
         level: {
           1: {
-            specialComments: 'all',
-            roundingPrecision: 6
```

Comment:
This has no effect right now since we set the precision in node-sass. And upstream on main where we switched to dart-sass, we decided to keep their default.